client.go

```diff
@@ -195,6 +195,30 @@ func (c *Client) LsSync(ctx context.Context, opts ...LsOption) ([]PinStatusGette
 	return res, <-errCh
 }
 
+// Manual version of Ls that returns a single batch of results and int with total count
+func (c *Client) LsBatchSync(ctx context.Context, opts ...LsOption) ([]PinStatusGetter, int, error) {
+	var res []PinStatusGetter
+
+	settings := new(lsSettings)
+	for _, o := range opts {
+		if err := o(settings); err != nil {
+			return res, 0, err
+		}
+	}
+
+	pinRes, err := c.lsInternal(ctx, settings)
+	if err != nil {
+		return res, 0, err
```

Comment:
nit: it's basically the same but I like to return `nil, 0, err` here because it makes it obvious that we're just returning zero values if we have an error.